hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/diskUsage/diskUsage.tsx

```diff
@@ -265,12 +265,9 @@ export class DiskUsage extends React.Component<Record<string, object>, IDUState>
 
   updateDisplayLimit(e): void {
     let res = -1;
-    if (e.key === 'all') {
-      res = Number.MAX_VALUE;
-    } else {
+    if (e.key) {
       res = Number.parseInt(e.key, 10);
     }
-
```

Comment:
Why do we need all of this? Can't we just set the `res` like this?
`res = Number.parseInt(e.key, 10);`

Reply:
@dombizita 
Thanks Done changes as suggested by you.

Reply:
> @smitajoshi12 are all comments addressed given by Zita ?

@devmadhuu 

Yes Devesh all comments addressed . you can merge.